### PR TITLE
Display Count of Service Recipients in PDF

### PR DESCRIPTION
**Pull request by DeepikaDK001**

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/Index.cshtml [hunk 1]
```diff
@@ -168,87 +173,87 @@
                     <nhs-page-title title="@title"
                                     caption="Order @Model.Order.CallOffId" />
 
-                    <pdf-summary-list>
-                        <pdf-summary-list-row label-text="Order ID">
+                    <nhs-summary-list>
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/Index.cshtml]: This is a PDF, so the PDF classes should be used

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/Index.cshtml [hunk 2]
```diff
@@ -268,15 +273,26 @@
                     <div style="page-break-after: always"></div>
                 }
 
-                    <h1>Items added to this order</h1>
+                    <h1>Service Recipients</h1>
+
+                    <nhs-summary-list>
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/Index.cshtml]: Same as above. There are PDF tags available which should be used

**Review by bjss-colin — CHANGES_REQUESTED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/OrderItem.cshtml [hunk 3]
```diff
@@ -1,176 +1,88 @@
...
-                    </td>
-                }
-            </tr>
-        }
-
-        @if (hasServiceRecipientQuantities
-            && Model.RolledUpRecipientsForItem.Count > 1)
+    <br />
+    <h4>Pricing and quantities</h4>
+    <table catches-errors="false">
```
Comment (ghost) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/OrderItem.cshtml]: `catches-errors` is not a valid attribute on a standard HTML `table` element.

**Review by bjss-colin — CHANGES_REQUESTED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/OrderItem.cshtml [hunk 4]
```diff
@@ -1,176 +1,88 @@
...
+{
 <h3>@Model.CatalogueItem.Name</h3>
+}
 
 @{
     var itemType = Model.CatalogueItem.CatalogueItemType.DisplayName();
     var orderItemCostBasis = Model.OrderItemPrice?.BillingPeriod?.Description() ?? string.Empty;
     var hasServiceRecipientQuantities = ((IPrice)Model.OrderItemPrice)?.IsPerServiceRecipient() ?? false;
     var tierCosts = ((IPrice)Model.OrderItemPrice)?.CostPerTierForBillingPeriod(Model.RolledUpTotalQuantity) ?? new List<PriceCalculationModel>();
     var tiers = Model.OrderItemPrice?.OrderItemPriceTiers?.OrderBy(x => x.LowerRange).ToList() ?? new List<OrderItemPriceTier>();
```
Comment (martinkimmings) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Views/OrderSummary/OrderItem.cshtml]: Looks like some of these aren't used anymore. Worth checking and removing any unused code.

**Review by imranbashir01 — APPROVED** [review]

**Review by bjss-colin — APPROVED** [review]

**Pull request merged**